.changeset/chilly-pugs-promise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs': patch
+---
+
+useRealTimers in onCssReady test case
```

Comment:
I think we should skip changesets for tests and minor changes. It'll make important changes more difficult to spot

Reply:
I have removed the changeset changes 👍🏼 ⚡